test/s3_test.erl

```diff
@@ -134,6 +129,10 @@ list_objects() ->
                   s3:list(bucket(), "", 10, "")).
 
 fold() ->
+        {ok, _} = s3:put(bucket(), "1/1", "foo", "text/plain"),
+        {ok, _} = s3:put(bucket(), "1/2", "foo", "text/plain"),
+        {ok, _} = s3:put(bucket(), "1/3", "foo", "text/plain"),
+        {ok, _} = s3:put(bucket(), "2/1", "foo", "text/plain"),
```

Comment:
Why these puts became necessary? The tests order doesn't seem to have changed.

Reply:
This test depended on the previous test putting objects to s3.